pyspark/bigdl/keras1/converter.py

```diff
@@ -403,16 +408,218 @@ def create_reshape(self, klayer, kclayer):
         blayer = BLayer.Reshape(klayer.target_shape, None)
         return blayer
 
+    def create_repeatvector(self, klayer, kclayer):
+        return BLayer.Replicate(n_features=klayer.n,
+                                n_dim=1,
+                                bigdl_type="float")
+
     def create_merge(self, klayer, kclayer):
         self.__check_is_share_weights(kclayer)
+        input_shape = klayer.get_input_shape_at(0)
         if klayer.mode == "concat":
-            blayer = BLayer.Concat(
-                 dimension = klayer.concat_axis,
-                 bigdl_type="float")
-        else:
-            raise Exception("We don't support % right now" % kclayer.mode)
+            blayer = BLayer.JoinTable(
+                dimension=klayer.concat_axis,
+                n_input_dims=len(input_shape[0]) - 1,
+                bigdl_type="float")
+        elif klayer.mode == "sum":
+            blayer = BLayer.CAddTable(
+                inplace=False,
+                bigdl_type="float")
+        elif klayer.mode == "mul":
+            blayer = BLayer.CMulTable(bigdl_type="float")
+        elif klayer.mode == "max":
+            blayer = BLayer.CMaxTable(bigdl_type="float")
+        # error in dot
+        elif klayer.mode == "dot":
```

Comment:
add TODO and throw exception here?

Reply:
Dot has some error; cos and ave seemingly not supported in BigDL

Reply:
Throw exception for dot for now. Fix it later.